slave-dotnet10/Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM openshift3/jenkins-slave-base-rhel7
```

Comment:
for now this needs to be openshift/jenkins-slave-base-rhel7 because of the way our internal CI system works.  we are working on cleaning this mess up, sorry.